### ISSUE TITLE
Use "requireOnDemand" for avatar_change

### ISSUE THREAD

**Issue by vladikoff** [mozilla/fxa-content-server#3015]
Will save us 32kb in the JS bundle, might be worth it. 

Ref https://github.com/mozilla/fxa-content-server/blob/2985ff9ff34d6fa81269d954eb49a71df7daf401/app/tests/spec/lib/require-on-demand.js
Ref https://github.com/mozilla/fxa-content-server/blob/a3559e5b560cb7b8c73b8e900576b7ccdeb615c9/app/scripts/views/settings/avatar_change.js#L127
Ref https://github.com/mozilla/fxa-content-server/blob/a3559e5b560cb7b8c73b8e900576b7ccdeb615c9/app/scripts/views/settings/avatar_change.js#L85

After and Before screenshot (main.js): 
![](http://v14d.com/i/55e466cb242fe.jpg)

The challenge is to make sure that the crop bundle is all packaged up into a single file.

**Comment by shane-tomlinson** [comment]
@vladikoff - what is the difference once gzipped (the important number)

**Comment by vladikoff** [comment]
BEFORE: 173.65KB gzipped (592.87KB uncompressed)

AFTER: 164.46KB gzipped (560.7KB uncompressed)

**Comment by shane-tomlinson** [comment]
Excellent. 9k isn't a ton, but still closer to where we want to be. I'd
like to see us under 150kb. Does the new loaded bundle include jquery UI?
IIUC, that is only needed (or was only needed before the new settings work)
for the profile editor.

**Comment by vladikoff** [comment]
> Does the new loaded bundle include jquery UI?
IIUC, that is only needed (or was only needed before the new settings work)
for the profile editor.

No, does not include jquery ui. jquery ui as you say is only needed for avatar_change

**Comment by vladikoff** [comment]
Merged into https://github.com/mozilla/fxa-content-server/issues/3242
